pkg/jx/cmd/context.go

```diff
@@ -116,11 +116,13 @@ func (o *ContextOptions) Run() error {
 		if err != nil {
 			return fmt.Errorf("Failed to update the kube config %s", err)
 		}
-		fmt.Fprintf(o.Out, "Now using namespace '%s' from context named '%s' on server '%s'.\n", info(ctx.Namespace), info(newConfig.CurrentContext), info(kube.Server(config, ctx)))
+		fmt.Fprintf(o.Out, "Now using namespace '%s' from context named '%s' on server '%s'.\n",
+			info(ctx.Namespace), info(newConfig.CurrentContext), info(kube.Server(config, ctx)))
```

Comment:
`o.Kube()` instead of `kube` here?

Reply:
not necessary (also not exported). The only methods that get exposed by the interface are `LoadConfig` and `UpdateConfig`. All the others (like this) just act on the config object passed in so no _need_ to extract them to the interface.